src/main/java/com/chargerrobotics/commands/shooter/HoodAutoCommand.java

```diff
@@ -9,21 +9,40 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import com.chargerrobotics.subsystems.LimelightSubsystem;
+import com.chargerrobotics.subsystems.ShooterHoodSubsystem;
 
 public class HoodAutoCommand extends CommandBase {
+  private final ShooterHoodSubsystem shooterSubsystem;
+  private boolean isOpening;
   /**
    * Creates a new HoodAutoCommand.
    * 
    * Using the distance the robot is from the target, set the hood angle.
    * f(distance)
    */
-  public HoodAutoCommand() {
+  public HoodAutoCommand(ShooterHoodSubsystem shooterSubsystem) {
+    this.shooterSubsystem = shooterSubsystem;
     // Use addRequirements() here to declare subsystem dependencies.
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    System.out.println("hood auto command init");
```

Comment:
I believe that we have more robust logging than this available that we should use here.

You can reference the `AutoDriveLinear.java` file to see how it is used.